queue/python/queue.py

```diff
@@ -0,0 +1,33 @@
+class Queue:
+    def __init__(self):
+        self.queue = []
+
+    def is_empty(self):
+        return self.queue == []
+
+    def enqueue(self, item):
+        self.queue.insert(0, item)
+
+    def dequeue(self):
+        return self.queue.pop()
+
+    def size(self):
+        return len(self.queue)
+    
+    def front(self):
+        return self.queue[self.size()-1] 
+
+
+def main():
+    q = Queue()
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpxvw_xy63/queue/python/queue.py
+++ b/tmp/tmpxvw_xy63/queue/python/queue.py
@@ -13,7 +13,7 @@
 
     def size(self):
         return len(self.queue)
-    
+
     def front(self):
         return self.queue[self.size()-1] 
 
```